### PR TITLE
Update README.md with information about Knative

### PR DESCRIPTION
**Pull request by rhuss**
* Added a little section about Knative and clarify that this can run on OpenShift and Kubernetes
* Update the NOTES section to the coloured format that is supported by GitHub via https://github.com/orgs/community/discussions/16925

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -101,9 +104,13 @@ Pre-built images for all of the applications in the system can be found at [`qua
 
 Deployment descriptors for these images are provided in the [`deploy/k8s`](deploy/k8s) directory. There are versions for [OpenShift](https://www.openshift.com), [Minikube](https://quarkus.io/guides/deploying-to-kubernetes#deploying-to-minikube), [Kubernetes](https://www.kubernetes.io), and [Knative](https://knative.dev).
 
+> [!NOTE]
+> The [Knative](https://knative.dev/docs/) variant can be used on any Knative installation that runs on top of Kubernetes or OpenShift. For OpenShift, you need "OpenShift Serverless" installed from the OpenShift operator catalog. Using Knative has the benefit that services are scaled down to zero replicas when they are not used.
```
Comment (edeandrea) [inline README.md:108]: Is there a way to link to docs for OpenShift serverless?

**Review by edeandrea — COMMENTED** [review]
Thanks @rhuss for this! I wasn't aware of the GitHub formatting for notes! I'll apply these to all of the other READMEs as well!

The only comment I have is if there is a way to link to documentation about OpenShift serverless?

File: README.md (hunk 1 above)
Comment (rhuss) [reply README.md]: Good point, I've added a link to the documentation.

**Comment by edeandrea** [comment]
@rhuss i think i’ve somehow made a mess of your PR…I tried to add a commit on top of yours (applying this to other READMEs for individual apps). I've attempted to fix this in #420. Can you take a look over there?

**Pull request closed without merging**